velox/row/UnsafeRowFast.cpp

```diff
@@ -192,6 +213,12 @@ int32_t UnsafeRowFast::serializeVariableWidth(
       memcpy(buffer, value.data(), value.size());
       return value.size();
     }
+    case TypeKind::HUGEINT: {
+      auto value = decoded_.valueAt<int128_t>(index);
+      int32_t size;
+      DecimalUtil::toByteArray(value, buffer, size);
```

Comment:
Why toByteArray takes 'size' as an argument? Wouldn't it be nicer to return 'size' instead?